authz-module/index.md

```diff
@@ -0,0 +1,204 @@
+---
+parent:
+  title: Understanding the Authz Module
+order: 0
+description: Use the Cosmos SDK authz module to grant authorizations from one account (the granter) to another account (the grantee).
+---
+
+# Authz Module
+
+The authz ("authorization") module enables a granter to grant authorizations to a grantee that allows the grantee to execute messages on behalf of the granter.
+
+## Use Authz to Grant Authorizations
+
+By implementing the [authz module](https://docs.cosmos.network/v0.43/modules/authz/), Cosmos SDK application developers give users the ability to grant certain privileges to other users. For example, a user might want another user to vote on their behalf and so, rather than giving the other user access to their account, the user would grant an authorization that allows the other user to execute `MsgVote` on their behalf.
+
+How users decide to use the authz module is up to them. In one case, a validator might want to create a separate account for voting in order to keep their validator key more secure. In another case, a DAO might want to distribute authorizations to members of the DAO in which case a multisg account would grant authorizations to individual accounts and members of the DAO would be able to execute messages without requiring signatures from all the other members.
+
+In this tutorial, you spin up a single node network using the simulation application in Cosmos SDK (`simapp`), grant an authorization to another account, and then execute a message on behalf of the granter as the grantee.
+
+## Requirements
+
+Before you start the tutorial, you need to install the `simd` binary.
+
+Clone the `cosmos-sdk` repository:
+```
+git clone https://github.com/cosmos/cosmos-sdk
+```
+
+Change directories and check out `v0.43.0`:
+```
+cd cosmos-sdk && git checkout v0.43.0
+```
+
+Install the `simd` binary:
+```
+make install
+```
+
+Check to make sure the installation was successful:
+```
+simd version
+```
+
+You should see `0.43.0` printed to the console.
+
+## Configuration
+
+<!-- TODO: update tip to use `unsafe-reset-all` with better user experience -->
+
+::: tip
+If you have used `simd` before, you may already have a `.simapp` directory in you home directory. If you would like to keep the previous data, either save the directory to another location or use the `--home` flag and specify a different directory for each command in the following instructions. If not, remove the previous directory (`rm -rf ~/.simapp`).
+:::
+
+Run the following commands to configure the `simd` binary.
+
+Set the chain ID:
+```
+simd config chain-id demo
+```
+
+Set the keyring backend:
+```
+simd config keyring-backend test
+```
+
+## Key Setup
+
+Create a couple test keys. You will use the `--recover` option so that the addresses used within the example commands below are consistent.
```

Comment:
```suggestion
Create a couple of test keys. Use the `--recover` option so that the addresses used in the following example commands are consistent.
```

Reply:
above, below are words that we all use... but let's instead use "following" for below, see the below entry https://developers.google.com/style/word-list#letter-b

Reply:
Thanks! I will try to make better use of the style guide.